xrtpy/response/channel.py

```diff
@@ -149,11 +143,7 @@ def entrancefilter_substrate(self) -> str:
     def entrancefilter_material(self):
         """Filter material on entrance filter."""
         return self._en_filter_data["MATERIAL"]
-    
-    @property
-    def entrancefilter_thickness(self):
-        """Filter material thickness on entrance filter."""
-        return self._en_filter_data["THICK"]
```

Comment:
@joyvelasquez — A question!  There were duplicates of `entrancefilter_thickness` in this class.  One returned a `Quantity` and the other returned a number.  This change is making it so that it returns the `Quantity` with units of Å rather than the number.  Is this going to cause any problems with the code that you're working on now?  If so, we can switch it back to the way it was.

Reply:
Good find @namurphy ! No, this change will not cause any harm to the current code. It would be good idea to change it back. I actually wanted the quantity in return, but we're going to switch it back.